src/configure_transform.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/python
+
+import os
+import re
+import sys
+
+define_re = re.compile("#define ([A-Z_]+) (.*)")
+
```

Comment:
ok, sounds good! I'm being a bit slow today, but will be on campus within an hour or so.